helm/templates/rbac.yaml

```diff
@@ -0,0 +1,80 @@
+{{- $functionNs := default .Release.Namespace .Values.functionNamespace }}
+{{- if .Values.rbac }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    app: {{ template "faas-netesd.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: faas-controller
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: faas-controller
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - get
```

Comment:
we should try hosting this in the values file to keep the chart as configurable as possible.

Reply:
you can always bind more roles to the `faas-controller` user, this is the minimum permissions to get faas-netes working

Reply:
@gardlt Can you help with testing?